lib/js/app/utils/exportToJson.ts

```diff
@@ -0,0 +1,24 @@
+import { download } from './download';
+
+const FILENAME = 'chart';
+
+export const exportToJson = ({
+  data,
+  fileName = FILENAME,
+}: {
+  data: Record<string, any>;
+  fileName: string;
+}) => {
+  let downloadFileName = fileName;
+  const hasExtension = fileName.includes('.json');
+
+  if (!hasExtension) {
+    downloadFileName = `${fileName}.json`;
+  }
+
+  try {
+    download(JSON.stringify(data), downloadFileName, 'text/json;charset:utf-8');
+  } catch (err) {
+    alert('There was a problem with JSON export');
+  }
```

Comment:
Please, remove try catch block and move this exception handler to `exportChartToJson` saga level. This will allow to show a toast notification about failure operation.